### PR TITLE
improve password-manager metadata attributes

### PR DESCRIPTION
**Pull request by kumavis**

### CODE REVIEW

**Comment by kumavis** [comment]
need to ensure `TextField` passes attributes through

File: ui/pages/first-time-flow/create-password/import-with-seed-phrase/import-with-seed-phrase.component.js [hunk 1]
```diff
@@ -301,7 +303,7 @@ export default class ImportWithSeedPhrase extends PureComponent {
             this.handleConfirmPasswordChange(event.target.value)
           }
           error={confirmPasswordError}
-          autoComplete="confirm-password"
+          autoComplete="new-password"
```
Comment (kumavis) [inline ui/pages/first-time-flow/create-password/import-with-seed-phrase/import-with-seed-phrase.component.js:306]: according to https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete confirming a new password should use the `new-password` value

**Comment by kumavis** [comment]
( ci test failure seems unrelated )

**Review by Gudahtt — APPROVED** [review]
LGTM!

**Review by danjm — APPROVED** [review]

**Pull request merged**